deploy/roles/athos/files/database.sql

```diff
@@ -7,27 +7,44 @@ GRANT ALL PRIVILEGES ON DATABASE dartagnan to dtuser;
 CREATE TABLE subscription_plans (
     id serial not null primary key,
     code character varying(1024) not null,
+    base_code character varying(1024) not null,
     name character varying(1024) not null,
     description character varying(1024) not null,
     price numeric,
-    period numeric default null
+    base_price numeric,
+    period numeric default null,
+    UNIQUE (id, code)
 );
 
-INSERT INTO subscription_plans VALUES (1, 'trial', 'Trial Pizza', '30 Day Trial', 0, 30);
-INSERT INTO subscription_plans VALUES (2, 'crostino', 'Crostino', '- Stable Updates repository
+INSERT INTO subscription_plans VALUES (1, 'trial', 'trial', 'Trial Pizza', '30 Day Trial', 0, 0, 30);
+INSERT INTO subscription_plans VALUES (2, 'crostino', 'crostino', 'Crostino', '- Stable Updates repository
```

Comment:
we should keep them in case there are systems with older versions. if we were to do a disaster recovery and re-import the db, these subscriptions must be there. we may remove them in the future when we are sure that everyone will only have the new ones.